docs/dev-environments/ethereum/besu/startup.sh

```diff
@@ -32,10 +33,21 @@ echo ""
 echo "========================================================="
 echo "STEP 3 :: Initialize truffle project and deploy contracts"
 echo "========================================================="
-docker-compose -f docker-compose-truffle.yaml up \
-|| error_exit "Failed to initialize truffle or deploy contracts"
+
+if [ -z "$TCF_HOME" ] || [[ `pwd` != "$TCF_HOME"* ]];
```

Comment:
kindly put [[ ]] around $TCF_HOME test condition!

Reply:
I think you can modify this condition like this:
-> test for if $TCF_HOME is set or not
       => if not, then throw error and exit
       => if yes, then change directory to $TCF_HOME
I don't think [[ `pwd` != "$TCF_HOME"* ]] check is needed.

Reply:
> kindly put [[ ]] around $TCF_HOME test condition!

Thanks. Updated.

Reply:
> I think you can modify this condition like this:
> -> test for if $TCF_HOME is set or not
> => if not, then throw error and exit
> => if yes, then change directory to $TCF_HOME
> I don't think [[ `pwd` != "$TCF_HOME"* ]] check is needed.

Clarified  the purpose here. need to run from the same directory but ensuring that it is the right directory and not outside `TCF_HOME`.

Reply:
these are not the steps to be done manually but will happen automatically through this script. So, I am suggesting, if TCF_HOME is set, then change directory to TCF_HOME in your script. then you don't need the check if pwd != "$TCF_HOME"

Reply:
@rranjan3 Thanks for the clarification. Its clear now. So, I approve the PR.